RangePicker/tests/RangePicker-specs.js

```diff
@@ -1,105 +1,117 @@
-import {mount} from 'enzyme';
-import {RangePicker, RangePickerBase} from '../RangePicker';
+import '@testing-library/jest-dom';
+import {render} from '@testing-library/react';
+import userEvent from '@testing-library/user-event';
 
-const tap = (node) => {
-	node.simulate('mousedown');
-	node.simulate('mouseup');
-};
-const decrement = (slider) => tap(slider.find('Button').last());
-const increment = (slider) => tap(slider.find('Button').first());
+import {RangePicker, RangePickerBase} from '../RangePicker';
 
 describe('RangePicker Specs', () => {
 	test('should render a single child with the current value', () => {
-		const picker = mount(
+		const {getByText} = render(
 			<RangePicker min={-10} max={20} value={10} />
 		);
 
-		const expected = '10';
-		const actual = picker.find('PickerItem').text();
+		const textContent = getByText('10');
+		const expected = 'item';
+		const actual = textContent.className;
 
-		expect(actual).toBe(expected);
+		expect(textContent).toBeInTheDocument();
+		expect(actual).toContain(expected);
 	});
 
 	test('should increase by step amount on increment press', () => {
-		const picker = mount(
+		const {getByText} = render(
 			<RangePicker min={0} max={100} defaultValue={10} step={1} noAnimation />
 		);
 
-		increment(picker);
+		userEvent.click(getByText('▶'));
 
-		const expected = '11';
-		const actual = picker.find('PickerItem').first().text();
+		const textContent = getByText('11');
```

Comment:
Same here. Rename the variable

Reply:
And any other place where you used `textContent`

Reply:
Done